scripts/apps/monitoring/directives/MonitoringGroup.js

```diff
@@ -45,6 +45,8 @@ export function MonitoringGroup(cards, api, authoringWorkspace, $timeout, superd
             var monitoring = ctrls[0];
             var projections = search.getProjectedFields();
 
+            var containerElem = elem.find('.stage-content');
```

Comment:
I would delete the new block on line 60 and instead write this like:
```js
var containerElem = monitoring.viewColumn ? $(document).find('.content-list') : elem.find('.stage-content');
```
It would be easier for readers to see the final outcome when this logic is put together imo